environment.yml

```diff
@@ -0,0 +1,26 @@
+name: snorkeling
+dependencies:
+- _license=1.1
```

Comment:
Its not needed from what I gathered. I ran `conda env export` and that was included in the dependencies.